src/main/java/org/folio/circulation/resources/TenantActivationResource.java

```diff
@@ -12,17 +14,15 @@
 import io.vertx.ext.web.Router;
 import io.vertx.ext.web.RoutingContext;
 
-import java.util.Map;
-
 public class TenantActivationResource {
 
   public void register(Router router) {
     RouteRegistration routeRegistration = new RouteRegistration("/_/tenant", router);
     routeRegistration.create(this::enableModuleForTenant);
-    routeRegistration.deleteAll(this::disableModuleForTenant);
+    routeRegistration.deleteAll(ctx -> noContent().writeTo(ctx.response()));
```

Comment:
Do we stiil have to implement `DELETE /_/tenant` if it does nothing?

Reply:
In the [RMB 32 upgrade notes](https://github.com/folio-org/raml-module-builder/blob/master/doc/upgrading.md#version-320) they provide the whole tenant interface snippet and it does contain DELETE endpoint. Personally I don't know what will happen if we remove it and whether Okapi is ok with it's absence.

Reply:
As far as I remember, mod-circulation did not have any implementation of TenantAPI at all before it was integrated with mod-pubsub, and Okapi did not mind. But that's just me being curious, I guess we can keep this empty DELETE implementation.

Reply:
> As far as I remember, mod-circulation did not have any implementation of TenantAPI at all before it was integrated with mod-pubsub and Okapi did not mind.

It did not and Okapi was ok with that because it made no claim to. Now that it does, it needs to fulfil the full Tenant interface even if it does not need to do anything in some situations.